README.md

```diff
@@ -8,12 +8,17 @@ Here you will find necessary steps and samples to get the app running on your ma
 
 ## Setup
 
-### Setting up Environment Variables for the sample **"web"** app
+### Setting up Environment Variables for the sample **"javascript"** or **"react"** app
 
 Create a `.env` file from `.env.example` in the web directory and fill out the environment variables.
 
 ```bash
-cp apps/web/.env.example apps/web/.env
+cp apps/javascript/.env.example apps/javascript/.env
```

Comment:
I meant to commit some symlinks inside of each of the apps to point at the top level `.env` but i'm realizing now they got ignored. 

Once those are updated we should be able to just have one .env at the top level with this command:
`cp .env.example .env`

Reply:
https://github.com/GetPortabl/backup-with-portabl-quickstart/pull/4